ui/selectors/selectors.js

```diff
@@ -490,6 +490,24 @@ export function getAllTokens(state) {
   return state.metamask.allTokens;
 }
 
+/**
+ * Get a flattened list of all ERC-20 tokens owned by the user.
+ * Includes all tokens from all chains and accounts.
+ *
+ * @returns Object[] - All ERC-20 tokens owned by the user in a flat array.
```

Comment:
Nit: JSDoc returns don't need a dash (the dash separates the name from the description, but there is no name here). The type should also be in curly braces. Looks like this was formatted with the type as the name by accident.

```suggestion
 * @returns {Object[]} All ERC-20 tokens owned by the user in a flat array.
```